src/domain/app/layout/utilityComponents/InfoTemplate.tsx

```diff
@@ -1,31 +1,40 @@
 import React from 'react';
+import classnames from 'classnames';
 
 import adultFaceIcon from '../../../../assets/icons/svg/adultFace.svg';
 import Icon from '../../../../common/components/icon/Icon';
 import Button from '../../../../common/components/button/Button';
 import styles from './infoTemplate.module.scss';
 
-type Props = {
+export type InfoTemplateProps = {
   title: string | React.ReactElement;
-  description: string | React.ReactElement;
-  icon?: string;
+  description?: string | React.ReactElement;
+  icon?: string | React.ReactElement;
   callToAction?: {
     label: string | React.ReactElement;
     onClick: () => void;
   };
+  classes?: string;
 };
 
 const InfoTemplate = ({
   title,
   description,
   callToAction,
   icon = adultFaceIcon,
-}: Props) => {
+  classes,
+}: InfoTemplateProps) => {
   return (
-    <div className={styles.infoPageLayout}>
+    <div className={classnames(styles.infoPageLayout, classes)}>
       <h1 className={styles.infoPageLayoutTitle}>{title}</h1>
-      <Icon className={styles.infoPageLayoutFace} src={icon} />
-      <p className={styles.infoPageLayoutDescription}>{description}</p>
+      {typeof icon === 'string' ? (
+        <Icon className={styles.infoPageLayoutFace} src={icon} />
+      ) : (
+        icon
+      )}
```

Comment:
Out of curiosity, was there something broken before? I don't see any changes in this PR using InfoTemplate with ReactElement, only string. So are you just making this more futureproof so to speak?

Reply:
Yep, "futureproof". I tested a HDS icon there, but wasn't happy with it.